PanzerChasm/client/map_drawer_gl.cpp

```diff
@@ -92,14 +92,19 @@ SIZE_ASSERT( FloorVertex, 4u );
 struct WallVertex
 {
 	unsigned short xyz[3]; // 8.8 fixed
+	unsigned short pad1;
 	short tex_coord[2]; // 8.8 fixed
 	unsigned char texture_id;
```

Comment:
"texture_id" is not 4-byte aligned. AMD driver does not require such alignment for this attribute?

Reply:
It should be, the tex_coord[2] array before it is 2*16 bit. I don't have any warnings or crashes with this change from the driver.

Reply:
If all works - fine.